CorsixTH/Lua/utility.lua

```diff
@@ -404,6 +404,15 @@ function serialize(val, options, depth, pt_reflist)
   end
 end
 
+--! Simplified interface for serializing a value with option for depth
+-- See serialize for further explanation
+--!param value A table or string to inspect
+--!param depth (num) Optional, defaults at 1
+--!return the serialized value as a console output
+function inspect(value, depth)
+  return print(serialize(value, {detect_cycles = true, max_depth = depth or 1, pretty = true}))
```

Comment:
`print` doesn't return anything, does it?

Reply:
I suppose you're right. Can you suggest a better way to rewrite the return param doc?

Reply:
Actually, return isn't needed and not sure why I put it in.

Reply:
Just don't write the `!return` line :)